### PR TITLE
Sorting day 00

### PR DESCRIPTION
**Pull request by davidabrahams**

### CODE REVIEW

File: sorting/day00/src/SortTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import com.sun.org.apache.regexp.internal.RE;
```
Comment (thecardkid) [inline sorting/day00/src/SortTest.java]: lol wtf is this import

**Comment by davidabrahams** [comment]
we weren't sure what happened with merge sort which is why we didn't have a hard sort here. will add quicksort. katie will write solutions.

File: sorting/day00/src/SortTest.java (hunk 1 above)
Comment (davidabrahams) [inline sorting/day00/src/SortTest.java]: uhhhhh. good question. idk. intellij did some shit.

**Comment by davidabrahams** [comment]
Addressed comments. Quicksort will be added to day02

**Pull request closed without merging**